controls/V-71849.rb

```diff
@@ -63,10 +55,38 @@
 command:
 
 #rpm --setugids <packagename>"
-
+  tag "fix_id": "F-78201r3_fix"
   # @todo add puppet content to fix any rpms that get out of wack
-  describe command("rpm -Va | grep '^.M' | wc -l") do
-    its('stdout.strip') { should eq '0' }
+# The following are known to be different and must be excluded. These are changed by the following
```

Comment:
This makes an assumption about system management that should not be made. Overrides should be made downstream by the user of the profile instead of hard coding for a particular point of view.